heliopy/data/ace.py

```diff
@@ -81,7 +81,7 @@ def mfi_h0(starttime, endtime):
 
     Returns
     -------
-    data : DataFrame
+    data : sunpy.timeseries.timeseriesbase.GenericTimeSeries
```

Comment:
These aren't showing up as links in the docs, they might need a :class: bit and ` around them.